src/main/java/com/google/flourbot/entity/action/SheetAppendAction.java

```diff
@@ -0,0 +1,36 @@
+package com.google.flourbot.entity.action;
+
+import java.util.Map;
+import com.google.flourbot.entity.action.Action;
+
+
+public final class SheetAppendAction extends Action {
```

Comment:
nit: when you add more actions consider making a parent of this class that extends Action and implements methods like getSheetUrl, which will be common for all sheet actions.

Reply:
Created #12 for this issue